src/evidently/ui/api/static.py

```diff
@@ -1,50 +1,42 @@
-import os
 import pathlib
-from typing import Optional
 
 import litestar
-from litestar import Litestar
 from litestar import MediaType
-from litestar import Request
 from litestar import Response
-from litestar.static_files import StaticFilesConfig
+from litestar.response import File
+from litestar.router import Router
+from litestar.static_files import create_static_files_router
 
+base_path = pathlib.Path(__file__).parent.parent.resolve() / "ui"
```

Comment:
This should probably be replaced with [`importlib.resources`](https://docs.python.org/3.11/library/importlib.resources.html#module-importlib.resources), but I wasn't sure how you're packacking this so I left it as is